app/views/posts/index.html.erb

```diff
@@ -23,7 +23,7 @@
             <span class="posts__post--likes">Likes: <%= post.likes_counter || 0 %></span>
           </div>
         </div>
-      </a>
+      <% end %>
 
       <ul class="posts__comment">
         Comments:
```

Comment:
- [ ] **[OPTIONAL]** I get this error page when I visit this page.

  ![image](https://github.com/Ogaga01/Blog-App/assets/36542986/f8e825bb-9ed5-465a-ac32-c09066cf2f33)

  To fix this, in the Post model change this line: `@comments.order(created_at: :desc).limit(5)` to this line:

  ```
  comments.order(created_at: :desc).limit(5)
  ```

  Next, you will get this error page.

  ![image](https://github.com/Ogaga01/Blog-App/assets/36542986/a765a22a-64a0-4980-a9db-49d8cd0a895e)

  You need to change the line 32 on this HTML file from this
  
  ```
  <li class="posts__comment--li"><%= comment.user.name %>: <%= comment.text %> </li>
  ```

  to this because a comment belongs to an `author`, not `user`:

  ```
  <li class="posts__comment--li"><%= comment.author.name %>: <%= comment.text %> </li>
  ```